expan/core/early_stopping.py

```diff
@@ -1,73 +1,76 @@
 
-    Returns:
-        float: redistributed alpha value at the time point with the given 
-               information fraction
+    :return: redistributed alpha value at the time point with the given information fraction
+    :rtype:  float
     """
     return (1 - norm.cdf(norm.ppf(1 - alpha / 2) / np.sqrt(information_fraction))) * 2
 
 
-def make_group_sequential(spending_function='obrien_fleming', estimated_sample_size=None, alpha=0.05, cap=8,
-                          multi_test_correction=False, num_tests=1):
+def make_group_sequential(spending_function='obrien_fleming', estimated_sample_size=None, alpha=0.05, cap=8):
+    """ A closure to the group_sequential function. """
     def f(x, y):
-        return group_sequential(x, y, spending_function, estimated_sample_size,
-                                alpha, cap, multi_test_correction, num_tests)
+        return group_sequential(x, y, spending_function, estimated_sample_size, alpha, cap)
     return f
 
 
-def group_sequential(x,
-                     y,
-                     spending_function='obrien_fleming',
-                     estimated_sample_size=None,
-                     alpha=0.05,
-                     cap=8,
-                     multi_test_correction=False,
-                     num_tests=1):
-    """
-    Group sequential method to determine whether to stop early or not.
-
-    Args:
-        x (array_like): sample of a treatment group
-        y (array_like): sample of a control group
-        spending_function: name of the alpha spending function, currently
-            supports: 'obrien_fleming'
-        estimated_sample_size: sample size to be achieved towards
-            the end of experiment
-        alpha: type-I error rate
-        cap: upper bound of the adapted z-score
-        multi_test_correction (boolean): flag of whether the correction for multiple testing is needed
-        num_tests (integer): number of tests or reported kpis used for multiple correction
-        
-    Returns:
-        EarlyStoppingStatistics object
+def group_sequential(x, y, spending_function='obrien_fleming', estimated_sample_size=None, alpha=0.05, cap=8):
+    """ Group sequential method to determine whether to stop early or not.
+
+    :param x: sample of a treatment group
+    :type  x: pd.Series or list (array-like)
+    :param y: sample of a control group
+    :type  y : pd.Series or list (array-like)
+    :param spending_function: name of the alpha spending function, currently supports: 'obrien_fleming'
+    :type  spending_function: str
+    :param estimated_sample_size: sample size to be achieved towards the end of experiment
+    :type  estimated_sample_size: int
+    :param alpha: type-I error rate
+    :type  alpha: float
+    :param cap: upper bound of the adapted z-score
+    :type  cap: int
+
+    :return: results of type EarlyStoppingTestStatistics
+    :rtype:  EarlyStoppingTestStatistics
     """
     # Checking if data was provided
     if x is None or y is None:
```

Comment:
This check doesn't seem to be needed, since the lines below ensure that `x` and `y` are instances of `Series`, `array` or `list` classes.

Reply:
Agree! Will remove that check.